2012-05-01  Landry Breuil  <landry@openbsd.org>

        [GTK] Fix gstreamer detection during configure
        https://bugs.webkit.org/show_bug.cgi?id=84325

        Reviewed by Philippe Normand.

        * configure.ac: Fix GSTREAMER_REQUIRED_VERSION and
        GSTREAMER_PLUGINS_BASE_REQUIRED_VERSION assignments.

Index: /trunk/configure.ac
===================================================================
--- configure.ac	(revision 115666)
+++ configure.ac	(revision 115737)
@@ -343,10 +343,10 @@
 
 case "$with_gstreamer" in
-     0.10) GSTREAMER_REQUIRED_VERSION=GSTREAMER_0_10_REQUIRED_VERSION
-           GSTREAMER_PLUGINS_BASE_REQUIRED_VERSION=GSTREAMER_0_10_PLUGINS_BASE_REQUIRED_VERSION
+     0.10) GSTREAMER_REQUIRED_VERSION=$GSTREAMER_0_10_REQUIRED_VERSION
+           GSTREAMER_PLUGINS_BASE_REQUIRED_VERSION=$GSTREAMER_0_10_PLUGINS_BASE_REQUIRED_VERSION
            GST_API_VERSION=0.10
            ;;
-     0.11) GSTREAMER_REQUIRED_VERSION=GSTREAMER_0_11_REQUIRED_VERSION
-           GSTREAMER_PLUGINS_BASE_REQUIRED_VERSION=GSTREAMER_0_11_PLUGINS_BASE_REQUIRED_VERSION
+     0.11) GSTREAMER_REQUIRED_VERSION=$GSTREAMER_0_11_REQUIRED_VERSION
+           GSTREAMER_PLUGINS_BASE_REQUIRED_VERSION=$GSTREAMER_0_11_PLUGINS_BASE_REQUIRED_VERSION
            GST_API_VERSION=0.11
            ;;
